packages/@dcl/inspector/src/components/App/vars.css

```diff
@@ -0,0 +1,19 @@
+:root {
```

Comment:
i believe this is still necessary to set the default values right? Otherwise this would only work when embedded in vscode

Reply:
juanca just answered this for me below 😝 

`i believe this is still necessary to set the default values right? Otherwise this would only work when embedded in vscode`